[2024-08-19T17:19:06.805+0500] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-08-19T17:19:06.844+0500] {taskinstance.py:2603} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: 3_models_dag.train_lr manual__2024-08-19T12:18:58.192332+00:00 [queued]>
[2024-08-19T17:19:06.854+0500] {taskinstance.py:2603} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: 3_models_dag.train_lr manual__2024-08-19T12:18:58.192332+00:00 [queued]>
[2024-08-19T17:19:06.855+0500] {taskinstance.py:2856} INFO - Starting attempt 1 of 1
[2024-08-19T17:19:06.870+0500] {taskinstance.py:2879} INFO - Executing <Task(PythonOperator): train_lr> on 2024-08-19 12:18:58.192332+00:00
[2024-08-19T17:19:06.888+0500] {logging_mixin.py:190} WARNING - /home/nicolaspc/anaconda3/envs/mlops_env/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=8403) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-08-19T17:19:06.889+0500] {standard_task_runner.py:72} INFO - Started process 8576 to run task
[2024-08-19T17:19:06.889+0500] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', '3_models_dag', 'train_lr', 'manual__2024-08-19T12:18:58.192332+00:00', '--job-id', '13', '--raw', '--subdir', 'DAGS_FOLDER/train_models-2.py', '--cfg-path', '/tmp/tmp0go1lgkn']
[2024-08-19T17:19:06.891+0500] {standard_task_runner.py:105} INFO - Job 13: Subtask train_lr
[2024-08-19T17:19:06.959+0500] {task_command.py:467} INFO - Running <TaskInstance: 3_models_dag.train_lr manual__2024-08-19T12:18:58.192332+00:00 [running]> on host nicolaspc-HP-Pavilion-Laptop-15-eh1xxx
[2024-08-19T17:19:07.064+0500] {taskinstance.py:3122} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Nicolas Pal' AIRFLOW_CTX_DAG_ID='3_models_dag' AIRFLOW_CTX_TASK_ID='train_lr' AIRFLOW_CTX_EXECUTION_DATE='2024-08-19T12:18:58.192332+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-08-19T12:18:58.192332+00:00'
[2024-08-19T17:19:07.065+0500] {taskinstance.py:731} INFO - ::endgroup::
[2024-08-19T17:19:07.095+0500] {s3.py:1387} INFO - Downloading source S3 file from Bucket test-bucket-nicolas-1 with path dataset/X_train.pkl
[2024-08-19T17:19:07.110+0500] {base.py:84} INFO - Retrieving connection 'aws_default'
[2024-08-19T17:19:07.247+0500] {credentials.py:1147} INFO - Found credentials in environment variables.
[2024-08-19T17:19:09.566+0500] {s3.py:1387} INFO - Downloading source S3 file from Bucket test-bucket-nicolas-1 with path dataset/X_test.pkl
[2024-08-19T17:19:10.041+0500] {s3.py:1387} INFO - Downloading source S3 file from Bucket test-bucket-nicolas-1 with path dataset/y_train.pkl
[2024-08-19T17:19:10.667+0500] {s3.py:1387} INFO - Downloading source S3 file from Bucket test-bucket-nicolas-1 with path dataset/y_test.pkl
[2024-08-19T17:19:11.092+0500] {train_models-2.py:116} INFO - Данные загружены
[2024-08-19T17:19:11.109+0500] {python.py:240} INFO - Done. Returned value was: {'start_tiemstamp': '20240819 17:19', 'lr_train_start': '20240819 17:19', 'lr_train_end': '20240819 17:19', 'lr_R^2_score': 0.575787706032451, 'lr_rmse': 0.7455813830127763, 'lr_mae': 0.4102330008496219}
[2024-08-19T17:19:11.161+0500] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-08-19T17:19:11.162+0500] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=3_models_dag, task_id=train_lr, run_id=manual__2024-08-19T12:18:58.192332+00:00, execution_date=20240819T121858, start_date=20240819T121906, end_date=20240819T121911
[2024-08-19T17:19:11.196+0500] {local_task_job_runner.py:261} INFO - Task exited with return code 0
[2024-08-19T17:19:11.221+0500] {taskinstance.py:3891} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-08-19T17:19:11.224+0500] {local_task_job_runner.py:240} INFO - ::endgroup::
